automation/project/Dependencies.scala

```diff
@@ -1,22 +1,25 @@
 import sbt._
+import sbt.Keys._
 
 object Dependencies {
-  val jacksonV = "2.9.5"
+  val jacksonV = "2.9.2"
   val akkaV = "2.5.7"
   val akkaHttpV = "10.1.1"
 
-  val serviceTestV = "0.8-c5b80d2"
+  val serviceTestV = "0.9-cbc35de"
   val workbenchModelV  = "0.11-2ce3359"
   val workbenchMetricsV  = "0.3-c5b80d2"
   val workbenchGoogleV = "0.16-c5b80d2"
 
   val workbenchModel: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-model" % workbenchModelV
   val workbenchMetrics: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-metrics" % workbenchMetricsV
 
+  val excludeV = scalaBinaryVersion
```

Comment:
nitpick: I think we can use `scalaBinaryVersion` directly instead of another variable